taiga2/models.py

```diff
@@ -1,7 +1,9 @@
 import enum
+from sqlite3 import Timestamp
 import uuid
 import re
 import datetime
+from xmlrpc.client import DateTime
```

Comment:
Is this import left here by mistake? I don't think you'd want to use that class for anything in Taiga...